### PR TITLE
20406-update-iceberg-to-v058

### PR DESCRIPTION
**Pull request by estebanlm**
https://pharo.fogbugz.com/f/cases/20406

### CODE REVIEW

**Comment by estebanlm** [comment]
case 20406

File: src/BaselineOfIDE.package/BaselineOfIDE.class/instance/groups..st [hunk 1]
```diff
@@ -209,7 +209,6 @@ spec group: 'Shout-Group' with: #(
 ).
 
 spec group: 'MetacelloExtras-Core' with: #(
-	'Metacello-GitBasedRepository'
```
Comment (guillep) [inline src/BaselineOfIDE.package/BaselineOfIDE.class/instance/groups..st]: This is wrong. Somehow the merge is removing this line and this line should stay.

File: src/BaselineOfIDE.package/BaselineOfIDE.class/instance/baseline..st [hunk 2]
```diff
@@ -66,8 +66,6 @@ baseline: spec
 		spec package: 'Zodiac-Extra'.	
 		spec package: 'Zinc-Zodiac'.	
 			
-		spec package: 'Metacello-GitBasedRepository'.
```
Comment (guillep) [inline src/BaselineOfIDE.package/BaselineOfIDE.class/instance/baseline..st]: This is wrong. Somehow the merge is removing this line and this line should stay.

**Comment by estebanlm** [comment]
yeah, that's weird... there should be just one method change.

**Comment by estebanlm** [comment]
et voilà, solved

**Comment by astares** [comment]
Cool! Thank you both !

**Pull request merged**